assets/scss/mq.scss

```diff
@@ -0,0 +1,8 @@
+@import '~sass-mq';
+
+$mq-responsive: true;
+
+$mq-breakpoints: (
+  desktop: 600px,
```

Comment:
I would say that 600px is too small to be "desktop" breakpoint. It's even small to be a "tablet" display, an iPad display is 768px, but we could call it like that.